analysis_options.yaml

```diff
@@ -0,0 +1,10 @@
+analyzer:
+  errors:
+    invalid_assignment: warning
+    missing_return: error
+    dead_code: info
```

Comment:
What is the default level for `dead_code` analysis? Wouldn't it be better to turn it up, maybe even to `error`? I have no experience with the kind of errors that Dart's DCA produces though, so it might be good as well.

Reply:
The default level for `dead_code` analysis is `warning`. I guess we can revert it. I've copied the configuration from Christina Lee https://twitter.com/RunChristinaRun/status/1080282186949906432
It was something that has loads of false positive (like [this one](https://github.com/dart-lang/sdk/issues/26673), and [this one](https://github.com/dart-lang/sdk/issues/34598)) but now it seems to be much better and lots of warnings have been fixed.